hydrolib/core/dflowfm/research/models.py

```diff
@@ -383,6 +388,35 @@ class Comments(Numerics.Comments):
             "Include density gradient in advection term (0: no(strongly advised), 1: yes, 2: Also in barotropic and baroclinic pressure term, 3,4: Also in vertical advection).",
             alias="jarhoxu",
         )
+        research_jaorgsethu: Optional[str] = Field(
+            "Deprecated variable.",
+            alias="jaorgsethu",
+        )
```

Comment:
This was considered obsolete and to not be implemented by @arthurvd, more info in https://github.com/Deltares/HYDROLIB-core/issues/648#issuecomment-2208515383